docker/matomo/Dockerfile

```diff
@@ -1,10 +1,8 @@
-FROM matomo:latest as gewisweb_matomo
+FROM matomo:4.10-fpm-alpine as gewisweb_matomo
```

Comment:
Is it desirable to pin Matomo to a specific version?

Reply:
Yes and no, I want to prevent the image from updating automatically to ensure we do not miss the update. Because we have had an update that broke the analytics for a week because we missed the update. This is mostly because of database updates that need to be performed.

I must admit that this will increase our maintenance workload, but that is still less than the time it takes to debug the instance when something breaks due to updates.